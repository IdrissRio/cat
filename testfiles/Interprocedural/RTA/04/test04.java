// test::q calls test04::m
public class test04 {
  void m(){};
}

class test extends test04 {
  void q() { m(); }
}
