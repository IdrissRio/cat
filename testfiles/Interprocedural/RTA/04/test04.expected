test04::Static:{
}
test04::Instance:{
}
test04::m:{
}
test::Static:{
}
test::Instance:{
}
test::q:{
	test04::m
}